### PR TITLE
no reciept notes for default_user

### LINKED ISSUE #1113: Should not invite default user to accept receipt note
I'm logged in as a default user.

From the Welcome page I can see a list of Pending Receipt Notes.  If I click on the title, I am shown the list.

If I select one, I am invited to mark `Receipt Note Received`.  If I do that, the backend rejects the operation, and I get logged out.

This is evidence that the backend security is working, but the front-end shouldn't offer the action to a `default-user`.

### PR DESCRIPTION
**Pull request by TahaKhanAbdalli**
fixes #1113

### CODE REVIEW

**Comment by TahaKhanAbdalli** [comment]
Okay  let me check.

**Comment by IanMayo** [comment]
Aah, I've found another issue @TahaKhanAbdalli 

We should disable the `print receipt` and `print hastener` buttons for default users.  They (mistakenly) try to update the `report printed` dates for the dispatch, which gets rejected by the backend, since this user doesn't have write access to dispatches.

File: src/resources/dispatch/DispatchList.tsx [hunk 1]
```diff
@@ -88,19 +90,19 @@ const BulkActions = (): React.ReactElement => {
       type: 'success'
     })
   }
-
+  const rolesThatCanEditPassword = ['rco-user', 'rco-power-user']
```
Comment (IanMayo) [inline src/resources/dispatch/DispatchList.tsx:93]: this const shoudl be named `rolesThatCanCreateReceiptNote`

File: src/resources/dispatch/DispatchShow.tsx [hunk 2]
```diff
@@ -147,50 +148,53 @@ const Footer = (props: FooterProps): React.ReactElement => {
   }
 
   if (typeof record === 'undefined') return <></>
-
+  const rolesThatCanEditPassword = ['rco-user', 'rco-power-user']
```
Comment (IanMayo) [inline src/resources/dispatch/DispatchShow.tsx:151]: This `const` needs to be renamed, too.

**Review by IanMayo — APPROVED** [review]
Tested and working

**Review by IanMayo — APPROVED** [review]

**Pull request merged**